modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -173,20 +169,18 @@ async function invokeServer<T>(
 					}),
 					method: 'POST',
 				})
-					.then((response) => response.text())
-					.then((text) => {
+					.then( async (response) => {
+					    if (!response?.ok) {
+					    	reject(`Error when fetching feature flag. HTTP Response code: ${response?.status}`)
```

Comment:
Why are we changing the error message? Before it was: `"Feature flag \"" + HtmlUtil.escape(key) + "\" is not available"`

Reply:
cool, I'll keep the same